Documentation/InstallGTM.rst

```diff
@@ -4,200 +4,271 @@
 .. role:: usertype
     :class: usertype
 
-The GT.M system is available for download from SourceForge_. This will download a compressed package which will need to be extracted before proceeding. The extraction can be in a location of your choice, as during the installation process you will be able to set where the installed files will be placed.  To have a common place where the GT.M files will be placed, we first create a folder within the /opt/ directory which will hold the files. The command used to create the folder is shown below.
+The GT.M binary and source code are available for download from SourceForge_.
+This tutorial will use the binary package for GT.M for Linux systems. There
+isn't much difference between the amd64 package and the x86 package as far as
+installation goes.
+
+Typically GT.M binary distributions follow the following convention:
+gtm_V{versionid}_{platform}_{arch}_pro.tar.gz. An example using the latest
+version of GT.M as of this writing is: ``gtm_V61000_linux_i686_pro.tar.gz``.
+
+The downloaded file is a compressed package which will need to be extracted
+before proceeding. The extraction can be in a location of your choice,
+as during the installation process you will be able to set where the installed
+files will be placed. We will follow the following convention for where to
+install GT.M: /usr/lib/fis-gtm/{Version}_{Platform}. We will use the example:
+``/usr/lib/fis-gtm/V61000_i686``
+
+To create this folder structure
 
 .. parsed-literal::
 
-  $ :usertype:`sudo mkdir /opt/gtm`
+  $ :usertype:`sudo mkdir -p /usr/lib/fis-gtm/V61000_i686`
```

Comment:
it might be better to specify the x86_64 version as i don't think 32bit platforms are used much anymore.